sound.js

```diff
@@ -84,6 +85,29 @@ function Sound(filename, basePath, onError, options) {
   });
 }
 
+Sound.prototype.addProgressListener = function({interval, callback}) {
```

Comment:
hey can we change this to 

```
addProgressListener((currentTime: number) => {
  
}, interval)
```

where the first argument and the second is the interval. Then it would match api's like `setInterval(() => {}, 10)` and `setTimeout(() => {}, 10)`